flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -613,25 +614,31 @@ protected void cleanUpInvoke() throws Exception {
 		// stop all timers and threads
 		tryShutdownTimerService();
 
+		Throwable suppressedThrowable = null;
 		// stop all asynchronous checkpoint threads
 		try {
 			cancelables.close();
 			shutdownAsyncThreads();
 		} catch (Throwable t) {
-			// catch and log the exception to not replace the original exception
-			LOG.error("Could not shut down async checkpoint threads", t);
+			// catch and suppress the exception to not replace the original exception
+			suppressedThrowable = ExceptionUtils.firstOrSuppressed(t, suppressedThrowable);
 		}
 
 		// we must! perform this cleanup
 		try {
 			cleanup();
 		} catch (Throwable t) {
-			// catch and log the exception to not replace the original exception
-			LOG.error("Error during cleanup of stream task", t);
+			// catch and suppress the exception to not replace the original exception
+			suppressedThrowable = ExceptionUtils.firstOrSuppressed(t, suppressedThrowable);
```

Comment:
ditto about the comment and losing error message.

Reply:
what is the suggestion?